test/spec/tokenManager.js

```diff
@@ -31,7 +31,8 @@ function createAuth(options) {
       storage: options.tokenManager.storage,
       storageKey: options.tokenManager.storageKey,
       autoRenew: options.tokenManager.autoRenew || false,
-      secure: options.tokenManager.secure // used by cookie storage
+      secure: options.tokenManager.secure, // used by cookie storage
+      tooManyRenewsSecondsWindow: options.tokenManager.tooManyRenewsSecondsWindow
```

Comment:
It doesn't look like this option is used?

Reply:
removed